README.md

```diff
@@ -54,6 +54,27 @@ CABOT_CAMERA_NAME_3
 CABOT_JETSON_CONFIG="D:192.168.1.51:rs1 D:192.168.1.52:rs2 D:192.168.1.53:rs3"
 ```
 
+## variable to power off remote hosts by the power off button
+```
+CABOT_REMOTE_POWEROFF_CONFIG
+# specify remote hosts in the order to power off by the power off button
+# write in the format <user name>@<host name>,
+# and separate config for multiple hosts by space
+# ex) "cabot@192.158.1.52 cabot@192.158.1.51 cabot@192.158.1.50"
```

Comment:
I would like to unify similar settings.
How about specifying `CABOT_JETSON_CONFIG` in order of shutting down sequence? and adding note in README file?
It does not affect to the launching procedure.

In addition to the ip addresses, the user name of the jetson is specified by 
`CABOT_JETSON_USER` in cabot and 
`CABOT_USER_NAME` in cabot-device-check. 
So, let's unify them to `CABOT_JETSON_USER`. I will fix cabot-device-check.

Please use `CABOT_ID_FILE`, `CABOT_ID_DIR` to specify identity file for ssh like cabot-device-check.
ttps://github.com/CMU-cabot/cabot-device-check/blob/0844a75fa46dae7f2d8f970de86358cea2f76773/check_device_status.sh#L120

Reply:
I modified to use `CABOT_JETSON_CONFIG `, `CABOT_JETSON_USER`, `CABOT_ID_FILE `, `CABOT_ID_DIR ` and tested with current version of cabot-device-check.

If it is better to test with updated version of cabot-device-check before merge, please update cabot-device-check.
I will test with updated version of cabot-device-check.

Reply:
I found cabot-device-check is already updated. I will test with new version.

Reply:
I modified to use updated cabot-device-check, and removed variable `CABOT_USER_NAME`.
Please review the changes.